src/reporter/index.js

```diff
@@ -174,14 +175,19 @@ export default class Reporter {
         task.on('test-run-start', async testRun => {
             const reportItem = this._getReportItemForTestRun(testRun);
 
+            reportItem.testRunIds.push(testRun.id);
```

Comment:
It would be nice if you provide a user agent info with `testRunId`

Reply:
the `onActionDone` method has `useragent`.